### ISSUE TITLE
[BUG]: Missing

### ISSUE THREAD

**Issue by KING009Ru** [typical-developers/oaklands-feedback#1181]
### Roblox Username

Minking250

### What version did this issue occur in?

1.69.2 or since market board update

### Describe the issue in detail.

Um there is Missing Ore types in market board i only know missing is Copper,Tin and more maybe if this is not on market board how we supposed to know their values? 

### Provide some images / videos of the issue in action.

![Screenshot 2024-11-09 191930](https://github.com/user-attachments/assets/32156591-f052-44e0-a37c-5cb05d970eed)
![Screenshot 2024-11-09 192015](https://github.com/user-attachments/assets/139846e0-8252-4d5b-bc39-d234ed6e73e8)
![Screenshot 2024-11-09 192035](https://github.com/user-attachments/assets/c9969e5c-3c36-44b6-9fc8-c16c60369649)
![Uploading Screenshot 2024-11-09 192053.png…]()

### Before Submitting

- [X] I have confirmed that nobody else reported this issue.
- [X] The details of this report are legitimate.
- [X] I understand that I cannot be reimbursed here for in-game content I have lost.

**Comment by KING009Ru** [comment]
![Uploading Screenshot 2024-11-09 192053.png…]()

**Comment by KING009Ru** [comment]
Under Crystal opal is Gold,Silver and magmite

**Comment by Imjustjokingwithya** [comment]
I don't think this is a bug, some common resources which aren't in the stock market board simply do not fluctuate in price. They basically have a fixed non changing price from my understanding.

**Comment by RedstoneRelic** [comment]
> I don't think this is a bug, some common resources which aren't in the stock market board simply do not fluctuate in price. They basically have a fixed non changing price from my understanding.

This is correct, and was stated by the devs to be the case
